apiClient/resource.go

```diff
@@ -39,6 +39,23 @@ func (client *Client) ReadResource(resourceAka string, properties map[string]str
 	return resource, nil
 }
 
+func (client *Client) ReadFullResource(resourceAka string) (*Resource, error) {
+	query := readFullResourceQuery(resourceAka)
```

Comment:
does readFullResourceQuery already exist?

Reply:
I think we should use `data` instead of `object` - then we would not need to remove the turbot property

Reply:
yes readFullResourceQuery existed but not used